db/migrate/20170913131914_create_active_admin_comments.rb

```diff
@@ -1,17 +0,0 @@
-class CreateActiveAdminComments < ActiveRecord::Migration::Current
-  def self.up
```

Comment:
Did you dropped the db and rerunned the migrations? the schema.rb file should be modified because of the deletion of this file